packages/protocol/engine/engine.go

```diff
@@ -333,12 +331,13 @@ func (e *Engine) initClock() {
 func (e *Engine) initTSCManager() {
 	e.TSCManager = tsc.New(e.Consensus.BlockGadget.IsBlockAccepted, e.Tangle, e.optsTSCManagerOptions...)
 
-	wp := e.Workers.CreatePool("TSCManager", 1) // Using just 1 worker to avoid contention
+	// wp := e.Workers.CreatePool("TSCManager", 1) // Using just 1 worker to avoid contention
```

Comment:
This is the orphanage stuff right? Should we remove it all as part of this PR? We can also get it back if we need, but at least we can get rid of all the orphanage-related events.

Reply:
Yeah, we could , but are we sure that we won't need those for transaction orphanage? I think we haven't concluded on what to do with it yet.